frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -38,6 +38,8 @@ export interface NewSurvey
         | 'targeting_flag'
         | 'archived'
         | 'appearance'
+        | 'created_at'
```

Comment:
curious, why does newSurvey type need created at and created by?

Reply:
It was asking for it before because of the way I destructured the survey object but now that I cleaned it up it's not necessary anymore